lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1805,11 +1803,10 @@ def test_failed_status_template(self, mock_sailthru_send, mock_log_error):
         """
         Test for failed verification.
         """
-        EmailMarketingConfiguration.objects.create(sailthru_verification_failed_template='test_template')
         data = {
             "EdX-ID": self.receipt_id,
             "Result": 'FAIL',
-            "Reason": 'Invalid photo',
+            "Reason": [{"photoIdReasons": ["Not provided"]}],
```

Comment:
Why this is changed?

Reply:
Reason value is now replaced by actual values that may come from software secure rather than using a dummy value.